innopoints/views/application.py

```diff
@@ -77,3 +92,85 @@ def take_back_application(project_id, activity_id):
         abort(400, {'message': 'Data integrity violated.'})
 
     return NO_PAYLOAD
+
+
+# ----- VolunteeringReport -----
+
+@api.route('/projects/<int:project_id>/activities/<int:activity_id>'
+           '/applications/<int:application_id>/report_info')
```

Comment:
Why the "info" part? The word "report" can be a noun as well. Let's keep it like the other endpoint

Reply:
That's the thing, it's not about just one report. I thought about how to best name this endpoint and came to this. This endpoint aggregates the info from all reports

Reply:
How can it be about all reports on an activity when you pass the application id? Or is it because multiple moderators can submit reports about a single volunteer?

Reply:
The whole chain of IDs here is merely for the purpose of identifying the person, on whom we are looking up reports, and the moderator set, whose reports we will consider